tests/features/custom_page/custom_page.search_field.feature

```diff
@@ -0,0 +1,38 @@
+@api
+Feature: "Custom page" search.
+  In order to use the search
+  As a user of the website
+  I need to be able to view search results in the "Custom page" canonical page.
```

Comment:
It is not the user of the website that wants this :D It's the collection owners that want to expose their content.

And nobody wants to view search results in the "Custom page" canonical page. We want to see search results in the search results page. For the end user this is just an overview of related content. They don't even know that this is built with a search engine in the backend. And they don't know what a canonical page is :)